lms/lmsweb/views.py

```diff
@@ -285,10 +330,15 @@ def user(user_id):
     if target_user is None:
         return fail(404, 'There is no such user.')
 
+    is_manager = current_user.role.is_manager
+
     return render_template(
         'user.html',
         solutions=Solution.of_user(target_user.id, with_archived=True),
         user=target_user,
+        role=current_user.role.name.lower(),
+        is_manager=is_manager,
```

Comment:
Prefer sending current_user instead these 2 parameters.

Reply:
The 'role' parameter - did it.
The 'is_manager' parameter - It's same as in "views".
It's instead of checking each time in jinja whether the `current_user` role is administrator or staff.